packages/@aws-cdk/toolkit-lib/lib/actions/drift/index.ts

```diff
@@ -0,0 +1,81 @@
+import type { StackSelector } from '../../api/cloud-assembly';
+
+export interface DriftOptions {
+  /**
+   * Criteria for selecting stacks to check for drift
+   */
+  readonly stacks: StackSelector;
+
+  /**
+   * Whether to fail with exit code 1 if drift is detected
+   *
+   * @default false
+   */
+  readonly fail?: boolean;
+}
+
+export interface FormattedDrift {
+  /**
+   * Resources that have not changed
+   */
+  readonly unchanged?: string;
+
+  /**
+   * Resources that were not checked for drift
+   */
+  readonly unchecked?: string;
+
+  /**
+   * Resources with drift
+   */
+  readonly modified?: string;
+
+  /**
+   * Resources that have been deleted (drift)
+   */
+  readonly deleted?: string;
+
+  /**
+   * The header, containing the stack name
+   */
+  readonly stackHeader?: string;
+
+  /**
+   * The final results (summary) of the drift results
+   */
+  readonly finalResult?: string;
+}
+
+export interface DriftResult {
+  /**
+   * Number of resources with drift
+   */
+  readonly numResourcesWithDrift?: number;
+
+  /**
+   * How many resources were not checked for drift
+   */
+  readonly numResourcesUnchecked?: number;
+
+  /**
+   * Complete formatted drift
+   */
+  readonly formattedDrift: FormattedDrift;
+}
+
+export interface DriftCommandResult {
```

Comment:
Looks like these are the same, so let's just use `DriftResult`

Reply:
Okay I see now `DriftResult` is an internal type and `DriftCommandResult` is public. 

Make sure `DriftResult` lives with the formatter (and probably rename it slightly). 

For the public type `DriftResult` is the better name.